dashboard/test/ui/features/feedback_tab.feature

```diff
@@ -3,55 +3,66 @@ Feature: Feedback Tab Visibility
 
   #As teacher, reviewing work, submit feedback
-  And I wait to see ".show-handle"
-  Then I click selector ".show-handle .fa-chevron-left"
-  Then I click selector ".section-student .name a"
-  And I press the first "#ui-test-feedback-input" element
-  And element "#ui-test-submit-feedback" contains text "Save and share"
-  And element "#ui-test-feedback-time" does not exist
-  And I press keys "Nice!" for element "#ui-test-feedback-input"
-  And I press "#ui-test-submit-feedback" using jQuery
-  And element ".editor-column" contains text "Nice!"
-  And I wait until "#ui-test-feedback-time" contains text "Last updated"
-  And element "#ui-test-submit-feedback" contains text "Update"
+    And I wait to see ".show-handle"
+    Then I click selector ".show-handle .fa-chevron-left"
+    Then I click selector ".section-student .name a"
+    And I press the first "#ui-test-feedback-input" element
+    And element ".editor-column" contains text "This is the key concept for this mini rubric."
+    And element "#ui-test-submit-feedback" contains text "Save and share"
+    And element "#ui-test-feedback-time" does not exist
+    And I press the first "#exceeds-input" element
+    And I press keys "Nice!" for element "#ui-test-feedback-input"
+    And I press "#ui-test-submit-feedback" using jQuery
+    And element ".editor-column" contains text "Nice!"
+    And element "#exceeds-input" is checked
+    And I wait until "#ui-test-feedback-time" contains text "Last updated"
+    And element "#ui-test-submit-feedback" contains text "Update"
 
   #As teacher, refresh page and latest feedback is visible
-  And I reload the page
-  And I wait for the page to fully load
-  And I wait until ".editor-column" contains text "Nice!"
-  And element ".editor-column" contains text matching "Last updated .* ago"
-  And element "#ui-test-submit-feedback" contains text "Update"
+    And I reload the page
+    And I wait for the page to fully load
+    And I wait until ".editor-column" contains text "Nice!"
+    And element "#exceeds-input" is checked
+    And element ".editor-column" contains text matching "Last updated .* ago"
+    And element "#ui-test-submit-feedback" contains text "Update"
 
   #As student, latest feedback from teacher is displayed
-  Then I sign out
-  And I sign in as "Lillian"
-  And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/7"
-  And I press the first ".uitest-feedback" element
-  And I wait until ".editor-column" contains text "Nice!"
-  And element ".editor-column" contains text matching "Feedback from Teacher_Lillian\(From .* ago\)"
+    Then I sign out
+    And I sign in as "Lillian"
+    And I am on "http://studio.code.org/s/allthethings/stage/38/puzzle/1"
+    And I press the first ".uitest-feedback" element
+    And I wait until ".editor-column" contains text "Nice!"
+    And element "#exceeds-input" is checked
+    And element ".editor-column" contains text matching "Last updated .* ago"
```

Comment:
Please check through this file and undo any whitespace-only changes that may have been automatically saved by your text editor, so that only lines which you've made substantial changes to show up as red or green